### DISCUSSION TITLE
useDialogPluginComposable

### DISCUSSION THREAD

**Discussion by omartb89** [quasarframework/quasar#11575] [General - Components / Directives / etc]
I got a copy of the quasar.dev's useDialogPluginComposable as a new component and saved it. I want to use it in a page. What stepsare vital in order to do that

**Comment by smolinari** [comment]
@omartb89 - Your question doesn't quite make sense. Is there something not clear here? 

https://quasar.dev/vue-composables/use-dialog-plugin-component

Otherwise, what is it you want to accomplish exactly in terms of a dialog?

Scott
